backend/app/services/adapters/task_kinds/queries.py

```
@@ -37,26 +43,140 @@
+            SELECT DISTINCT k.id, k.json
+            FROM tasks k
+            LEFT JOIN resource_members tm ON k.id = tm.resource_id AND tm.resource_type = 'Task' AND tm.user_id = :user_id AND tm.status = 'approved'
+            WHERE k.kind = 'Task'
+            AND k.is_active = true
+            AND k.namespace != 'system'
+            AND (k.user_id = :user_id OR tm.id IS NOT NULL)
+            ORDER BY k.created_at DESC
         """
+        )
+
         task_ids, total = get_accessible_task_ids_and_total(
```

Ruff (0.15.5)

[warning] Unpacked variable `task_ids` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)